### PR TITLE
Make go vet happy with server example

### PR DESCRIPTION
**Pull request by lutzhorn**
More changes to `README.md`:

`go vet` asks for comments on exported methods when run against the server code from the  `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,25 +61,29 @@ import (
 	"io/ioutil"
 	"log"
 
-	"github.com/emersion/go-smtp"
+	smtp "github.com/emersion/go-smtp"
```
Comment (emersion) [inline README.md]: Is this needed by `go vet`?

Comment (lutzhorn) [reply README.md]: Probalby not. This change was automatically made by the Go tools used by Visual Studio Code.

Comment (emersion) [reply README.md]: Can you remove this change and rebase your PR?

**Comment by lutzhorn** [comment]
Yes, I will undo this change.

**Comment by emersion** [comment]
Thanks!

**Pull request merged**